packages/graphql/src/schema-model/parser/annotations-parser/jwt-payload-annotation.ts

```diff
@@ -19,7 +19,6 @@
 import type { DirectiveNode } from "graphql";
 import { JWTPayloadAnnotation } from "../../annotation/JWTPayloadAnnotation";
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 export function parseJWTPayloadAnnotation(_directive: DirectiveNode): JWTPayloadAnnotation {
```

Comment:
Why do we have this argument at all? Seems totally bonkers.

Reply:
It is simply consistency, all the parsers receive the directive itself. 
We may want to change it, but it should be made across all parsers in the schema model